src/hotspot/share/runtime/continuation.hpp

```diff
@@ -47,7 +47,7 @@ void continuations_init();
 class javaVFrame;
 class JavaThread;
 
-// should match Continuation.toPreemptStatus() in Continuation.java
+// should match Continuation.PreemptStatus() in Continuation.java
```

Comment:
As far as I can tell, these enum values still don't match the Java values.  If they need to match, then maybe there should be asserts that check that.

Reply:
`PreemptStatus` is meant to be used with `tryPreempt()` which is not implemented yet, i.e. there is no method yet that maps between these values and the PreemptStatus enum. The closest is `Continuation.pinnedReason` which we do use. So if you want I can remove the reference to PreemptStatus and use pinnedReason instead.

Reply:
Yes, that would be better for now.

Reply:
Changed.